modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -598,9 +598,7 @@ definition {
 				kebabOption = "Delete",
 				objectLabel = "Entry Test B");
 
-			AssertTextEquals(
-				locator1 = "Message#SUCCESS_DISMISSIBLE",
-				value1 = "Success:Your request completed successfully.");
```

Comment:
This was a try to fix it, but not work, because the failure is before this. I replaced it with `Refresh()`; to guarantee that the entry was removed.

Reply:
Does this happen locally? If so, this is a bug. If not, then I'm OK with this as a workaround.

Reply:
No, no, I think I left a confusing message above.

This step works fine, but isn't necessary to cover, it was changed recently to try to fix the test.

Reply:
cc: @timpak